lxc: linux Container library

Authors:
Daniel Lezcano <daniel.lezcano@free.fr>

This library is free software; you can redistribute it and/or
modify it under the terms of the GNU Lesser General Public
License as published by the Free Software Foundation; either
version 2.1 of the License, or (at your option) any later version.

This library is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
Lesser General Public License for more details.

ou should have received a copy of the GNU Lesser General Public
License along with this library; if not, write to the Free Software
Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA

Date: Nov 27 10:50:30 2017

From: 0cadb71747b08193f24d2c501b66488024bfc071 Jan 15 16:38:29 2018

Subject: [PATCH] RDKB-14741: Remove lighttpd hook from meta-rdk

	Remove lighttpd hook from meta-rdk.

Source: COMCAST

Upstream-Status: Pending

Signed-off-by: Deva Thiyagarajan <Deva_Thiyagarajan@comcast.com>
---

Index: lxc-1.1.2/hooks/Makefile.am
===================================================================
--- lxc-1.1.2.orig/hooks/Makefile.am
+++ lxc-1.1.2/hooks/Makefile.am
@@ -5,6 +5,7 @@ hooks_SCRIPTS = \
 	mountecryptfsroot \
 	ubuntu-cloud-prep \
 	squid-deb-proxy-client \
-	rdksystemdhook
+	rdksystemdhook \
+	lighttpd_systemdhook
 
 EXTRA_DIST=$(hooks_SCRIPTS)
